packages/supercollider/build.sh

```diff
@@ -0,0 +1,20 @@
+TERMUX_PKG_HOMEPAGE=http://supercollider.github.io/
+TERMUX_PKG_DESCRIPTION="A platform for audio synthesis and algorithmic composition"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="@rene-descartes2021"
+TERMUX_PKG_VERSION=3.14.0-rc1
+TERMUX_PKG_SRCURL=https://github.com/supercollider/supercollider/releases/download/Version-${TERMUX_PKG_VERSION}/SuperCollider-${TERMUX_PKG_VERSION}-Source.tar.bz2
+TERMUX_PKG_SHA256=da5c0642f6d18b288539c600603be29e6e00d609bc664e2147bde146dabd6845
+TERMUX_PKG_DEPENDS="jack2, libyaml-cpp, fftw, readline, libsndfile"
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
+-DBUILD_SHARED_LIBS=ON
+-DNATIVE=ON
```

Comment:
It looks like this is causing build failures.
Our build system should should take care of setting the correct `-march=` automatically.
```suggestion
```